components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
```

Comment:
`contentView` already has an autoresizingMask set. You should not need to adjust its frame on each layout pass.

Reply:
Addressed in 808dd2535cf1f193b23024da6c1c20b98092bed9.